scripts/gopath.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+#  with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+export TOPWD=$(pwd)
+export BUILDDIR="$(mktemp -d)"
+export GOPATH="${BUILDDIR}"
+export SRCPATH="${BUILDDIR}/src/github.com/awslabs/amazon-ecr-credential-helper"
+declare -a LINKS=("ecr-login" "scripts" "Makefile" "VERSION" ".git")
+mkdir -p "${SRCPATH}"
+for LINK in "${LINKS[@]}"; do
+	ln -s "${TOPWD}/${LINK}" "${SRCPATH}"
+done
+cd "${SRCPATH}"
+ls
+source "${SRCPATH}/scripts/shared_env"
```

Comment:
Is this still necessary? It looks to me like the variables there are not appropriate for the version of Go used nor for the usage of tool on the path for build.

Reply:
Done